design/sketch.webidl

```diff
@@ -286,7 +286,7 @@ enum GPUBindingType {
 
 dictionary GPUBindGroupLayoutBinding {
     u32 binding;
-    GPUShaderStageFlags visibility;
+    GPUShaderStageFlags visibility = GPUShaderStageBit.VERTEX | GPUShaderStageBit.FRAGMENT | GPUShaderStageBit.COMPUTE;
```

Comment:
How often is an app really going to want to use a bind group in multiple stages? It seems better to have no default here.

I don't know if this causes deoptimizations.

Reply:
Ditto.

Reply:
It would be clean from the API standpoint of *all* flag values defaulted to zero.

Reply:
In Vulkan, per-stage bindings do not exist and must be emulated by the implementer. So this makes some sense -- defaulting to all stages possibly makes the Vulkan implementation easier.

Reply:
+1 to having a sentinel "all" value. We've been [bitten](https://github.com/w3c/csswg-drafts/issues/1641) by this in CSS before.

Reply:
@magcius not sure what you mean. There is a `VkDescriptorSetLayoutBinding::stageFlags` field in Vulkan for those flags.

Reply:
Today we resolved to make this required.